asterius/rts/rts.unicode.mjs

```diff
@@ -0,0 +1,66 @@
+export class Unicode {
+    constructor(logger) {
+        Object.seal(this);
+    }
+
+    u_gencat(code) {
+        if (code >= 'A'.charCodeAt(0) && code <= 'Z'.charCodeAt(0)) {
```

Comment:
In general, please use `codePointAt(0)` for converting a single-char js string to `Char#`, and `String.fromCodePoint()` for vice versa. Since the semantics of Haskell `Char#` is unicode code point.

Reply:
I'll change the code, but I think this is moot --- none of this code will work if the value held by a `Char#` is not ASCII...